io.github.ad_on_is.Resticity.yml

```diff
@@ -0,0 +1,71 @@
+id: io.github.ad_on_is.Resticity
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+
+
+command: resticity
+finish-args:
+  - --share=ipc # Share IPC namespace with the host (necessary for X11).
+  - --socket=x11
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri # OpenGL rendering support.
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
Is this needed? It should go through the portal if your toolkit supports it.

Reply:
Lol.. IDK... I just copy and paste stuff from across the internet